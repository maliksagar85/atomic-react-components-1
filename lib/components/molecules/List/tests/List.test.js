import React from 'react';
import { shallow } from 'enzyme';

import { ListVanilla } from '../index';

describe('<List />', () => {
  let ListComponent = '';
  beforeEach(() => {
    ListComponent = shallow(
      <ListVanilla>
        <li>This is list item</li>
        <li>This is list item</li>
      </ListVanilla>
    );
  });

  test('should render default view correctly', () => {
    expect(ListComponent).toMatchSnapshot();
  });
});
