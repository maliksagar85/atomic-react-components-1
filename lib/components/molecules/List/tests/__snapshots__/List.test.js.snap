// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<List /> should render default view correctly 1`] = `
<ul
  borderSeparator={false}
  className=""
  inheritedStyles=""
  inline={false}
  nested={false}
  noDefaultView={false}
  noMargin={false}
>
  <li>
    This is list item
  </li>
  <li>
    This is list item
  </li>
</ul>
`;
